test/helpers/deploy/deploy_resources.go

```diff
@@ -243,6 +255,49 @@ func KonnectGatewayControlPlaneWithID(
 	return cp
 }
 
+// KonnectCloudGatewayDataPlaneGroupConfiguration deploys a
+// KonnectCloudGatewayDataPlaneGroupConfiguration resource and returns the resource.
+func KonnectCloudGatewayDataPlaneGroupConfiguration(
+	t *testing.T,
+	ctx context.Context,
+	cl client.Client,
+	opts ...ObjOption,
+) *konnectv1alpha1.KonnectCloudGatewayDataPlaneGroupConfiguration {
+	t.Helper()
+	obj := konnectv1alpha1.KonnectCloudGatewayDataPlaneGroupConfiguration{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: "data-plane-group-configuration-" + uuid.NewString()[:8],
+		},
+		Spec: konnectv1alpha1.KonnectCloudGatewayDataPlaneGroupConfigurationSpec{
+			Version:   "3.9",
```

Comment:
Should we have it hardcoded here? maybe we can use `DefaultDataPlaneTag` constant that it is kept up to date by renovate. WDYT?

Reply:
👍 757be758dae247a66aa84c45d4da65353fbc2abe